vpr/src/draw/intra_logic_block.cpp

```diff
@@ -38,7 +38,7 @@
 
 static void draw_internal_load_coords(int type_descrip_index, t_pb_graph_node* pb_graph_node, float parent_width, float parent_height);
 static int draw_internal_find_max_lvl(const t_pb_type& pb_type);
-static void draw_internal_calc_coords(int type_descrip_index, t_pb_graph_node* pb_graph_node, int num_pb_types, int type_index, int num_pb, int pb_index, float parent_width, float parent_height, float* blk_width, float* blk_height);
+static void draw_internal_calc_coords(int type_descrip_index, t_pb_graph_node* pb_graph_node, int num_block, int num_columns, int num_rows, float parent_width, float parent_height, float* blk_width, float* blk_height);
```

Comment:
Add a doxygen comment documenting the overall functionality (what does this set up?  when should it be called) and all parameters (with @param).

Not essential for this PR, but we could change blk_width and blk_height to vectors and that would be nicer.

I think num_columns and num_rows are the number for drawing things inside this type of tile.  That needs to be explained. At first I thought they were from the grid (they aren't!).